### ISSUE TITLE
NVMF target perf test

### ISSUE THREAD

**Issue by alekseymmm** [spdk/spdk#107]
I'm trying to test nvmf target using spdk/examples/perf/perf utility.  I have 2 nodes, first one runs target with this config

[Global]
  ReactorMask 0xFFFFFFFF
  LogFacility "local7"

[Rpc
  Enable No
  Listen 127.0.0.1

[Nvmf]
  MaxQueuesPerSession 16
  MaxQueueDepth 128
  AcceptorCore 8
  AcceptorPollRate 10000

[Subsystem1]
  NQN nqn.2016-06.io.spdk:cnode11_1
  Core 9
  Mode Direct
  Listen RDMA 10.30.0.11:4420
  NVMe 0000:81:00.0

[Subsystem2]
  NQN nqn.2016-06.io.spdk:cnode11_2
  Core 10
  Mode Direct
  Listen RDMA 10.20.0.11:4420
  NVMe 0000:82:00.0

On the second node I run 
`spdk/examples/nvme/perf/perf -w randread -q 32 -s 4096 -t 5 -r 'trtype:RDMA adrfam:IPv4 traddr:10.30.0.11 trsvcid:4420 subnqn:nqn.2016-06.io.spdk:cnode11_1' `
EAL: Detected 32 lcore(s)
EAL: Auto-detected process type: PRIMARY
EAL: No free hugepages reported in hugepages-1048576kB
EAL: Probing VFIO support...
EAL: PCI device 0000:81:00.0 on NUMA socket 1
EAL:   probe driver: 1c58:3 spdk_nvme
EAL: PCI device 0000:82:00.0 on NUMA socket 1
EAL:   probe driver: 1c58:3 spdk_nvme
Initializing NVMe Controllers
Attaching to NVMe over Fabrics controller at 10.30.0.11:4420: nqn.2016-06.io.spdk:cnode11_1
nvme_ctrlr.c: 849:nvme_ctrlr_configure_aer: ***ERROR*** nvme_ctrlr_cmd_set_async_event_config failed!
Attached to NVMe over Fabrics controller at 10.30.0.11:4420: nqn.2016-06.io.spdk:cnode11_1
Associating HUSPR3216AHP301      (STM0001B5503        ) with lcore 0
Initialization complete. Launching workers.
Starting thread on core 0

Device Information                                     :       IOPS       MB/s    Average        min        max
HUSPR3216AHP301      (STM0001B5503        ) from core 0:  231593.20     904.66     138.12      16.68    1254.84
Total                                                  :  231593.20     904.66     138.12      16.68    1254.84

I am just curious whether this ***Error*** is important.  
The second question is should I try connecting to target nqn directly or use discovery service nqn or no nqn at all?

**Comment by optimistyzy** [comment]
For the second question, both can be OK. If your provide no nqn or discovery service nqn, you can get two subsystems with spdk nvme perf.  If you specify the nqn (not the discovery service nqn),  you can connect the subsystem with the nqn you provided.

**Comment by optimistyzy** [comment]
"nvme_ctrlr.c: 849:nvme_ctrlr_configure_aer: ERROR nvme_ctrlr_cmd_set_async_event_config failed!" It means that AER(ADVANCED ERROR reporting functionality) is not supported by nvmf target, we may improve it in current SPDK NVMF target.  It currently seems not influences the normal read write functions.

**Comment by alekseymmm** [comment]
Ok. Thanks a lot for the information. I run the same perf command on the nvmf target machine and got no errors and no warnings, just normal output. 
I have noticed that I got this messages on nvmf target server when  I had started perf that leads to 
nvme_ctrlr.c: 849:nvme_ctrlr_configure_aer: ERROR nvme_ctrlr_cmd_set_async_event_config failed!

`SET FEATURES (09) sqid:0 cid:57 nsid:0 cdw10:0000000b cdw11:0000001f`
`INVALID FIELD (00/02) sqid:0 cid:57 cdw0:0 sqhd:000e p:1 m:0 dnr:0`

May be this can help you. If this error is not crucial then I can continue my work because all reads and writes are fine.
